### ISSUE TITLE
No Enabled in examples

### ISSUE THREAD

**Issue by Multipanda** [MicrosoftDocs/windows-powershell-docs#2635]
The examples are all missing the Enabled parameter, as well as a password parameter. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1fb61649-0f07-ff16-3fbe-4029b9119c7e
* Version Independent ID: 5f3c184f-42bd-176e-5764-5411c3227c6e
* Content: [New-ADUser (ActiveDirectory)](https://docs.microsoft.com/en-us/powershell/module/activedirectory/new-aduser?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/activedirectory/New-ADUser.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/activedirectory/New-ADUser.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

**Comment by msbemba** [comment]
@Multipanda Thank you for your feedback. We have made changes to the document via PR#https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2809

Thanks
Sri

**Comment by Multipanda** [comment]
> @Multipanda Thank you for your feedback. We have made changes to the document via PR##2809
> 
> Thanks Sri

Great to see, thank you.
Though it seems the merger was blocked, though I guess we just have to wait for the right person to see it and confirm the merger, or is it something you can do? Not sure how such things work.

**Comment by msbemba** [comment]
@Multipanda That is correct. The merger was blocked as it needs approval from the author. We have the details in the document now.

@e0i This can be closed.

Thanks
Sri
